web/packages/teleport/src/Roles/RoleEditor/StandardEditor.tsx

```diff
@@ -337,6 +353,18 @@ export const StandardEditor = ({
                 validation={validation.rules}
               />
             </div>
+            <div
+              id={optionsTabId}
+              style={{
+                display: currentTab === StandardEditorTab.Options ? '' : 'none',
+              }}
+            >
```

Comment:
@avatus Why?

Reply:
Seems to me that we use Box everywhere as convention instead of div. I understand there are consistency issues with this but I'm hoping we don't add more.

Reply:
Alright, moving to Box.